config/routes.rb

```diff
@@ -147,7 +147,7 @@
       json_api_resources :subject_workflow_statuses, only: [:index, :show]
 
       json_api_resources(:translations, {
-        constraints: Routes::Constraints::Translations.new,
+        constraints: Routes::Constraints::TranslationsConstraint.new,
```

Comment:
Originally i felt the `Routes::Constraints::TranslationsConstraint` had the `Constraints` prefix in the module name so didn't need the extra wording as a suffix. I'm about 2/10 on this change.

Reply:
It confused the rails autoloader when I added the Translations::* namespace, and somehow Rails was expecting to find the `::Translations`` namespace to be defined in this file. Not sure why, but this was the simplest fix.